internal/storage/v2/grpc/proto/storage.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+package jaeger.storage.v2;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+import "opentelemetry/proto/trace/v1/trace.proto";
+
+option go_package = "storage_v2";
+
+// GetTraceRequest represents a query to get a single trace from the storage backend.
+message GetTraceRequest {
+    // trace_id is a 16 byte hex-encoded string containing the unique 
+    // identifier for the trace to query. 
+    string trace_id = 1;
```

Comment:
@yurishkuro should this be a string? or should we make it bytes?

Reply:
its bytes in `trace.proto`, so I think that would make sense here as well?

```proto
// A Span represents a single operation performed by a single component of the system.
//
// The next available field id is 17.
message Span {
  // A unique identifier for a trace. All spans from the same trace share
  // the same `trace_id`. The ID is a 16-byte array. An ID with all zeroes OR
  // of length other than 16 bytes is considered invalid (empty string in OTLP/JSON
  // is zero-length and thus is also invalid).
  //
  // This field is required.
  bytes trace_id = 1;
```

Reply:
yes. `bytes`